README.md

```diff
@@ -297,6 +297,7 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
 We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
 
 + Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the IP of the edge node
++ Make sure role is set to edge for the node. For this a key of the form `"node-role.kubernetes.io/edge"` must be present in `labels` tag of `metadata`. If role is not set for the node pods, configmap and secrets cannot sync with the node.
```

Comment:
If the "node-role.kubernetes.io/edge" label is required, we should add validation for this otherwise it's confusing.

Reply:
In the default node.json we have added this label. And we have validation in source code against this so that we do not consider master and other nodes of an existing k8s setup

Reply:
> In the default node.json we have added this label

People may don't always use default node.json to set up nodes, right?

> And we have validation in source code against this so that we do not consider master and other nodes of an existing k8s setup

Can you please show me this piece of code?

Reply:
@m1093782566 , any suggestions on how to ensure people have created node with correct role if they are not using our node.json ?

Reply:
I am thinking developing a installer tool like kubeadm to ease the pain of installation and improve the user experience.

Reply:
@m1093782566 , thanks for creating an issue for installer requirement

Reply:
yes , an installer should take care of such steps . thanks dujun for bringing this up.